README.md

```diff
@@ -7,7 +7,7 @@ This is a wrapper around the `google-cloud-pubsub` gem. There are times when it
 Add this line to your application's Gemfile:
 
 ```ruby
-gem 'pubsub_client'
+gem 'pubsub_client', github: 'apartmentlist/pubsub_client', tag: '<ADD-LATEST-TAG-HERE>'
```

Comment:
You needed to specify the source and tag to pull?

Reply:
Yes

Reply:
Which app? I just tried it in `api` and published from a Rails console successfully. I did not add the source+tag in the Gemfile.

Reply:
When I added just `gem 'pubsub_client'` to `api` and did a bundle install it could not find the gem. If you look in `locations`, it also specifies the `url` and `tag` in its `Gemfile`. Also, if you look in any repo that uses `dog_trainer`, all of those gem definitions include the `url` and `tag` as well.